install_config/upgrading/manual_upgrades.adoc

```diff
@@ -708,6 +708,41 @@ upgrade will fail and should be restarted automatically. This will not disrupt
 pods that are already running.
 ====
 
+ifdef::openshift-origin,openshift-enterprise[]
```

Comment:
That's fine. Probably don't need this `ifdef` here even (and its closing `endif`), since this topic only shows up in those two distros anyway. My ifdef comment was mainly about the version number below, which you've conditionalized now.